### ISSUE TITLE
Creating multiple instances of UI panels

### ISSUE THREAD

**Issue by aurora-1-2-3** [JannisX11/blockbench#2545]
### Detailed description of your suggestion

After right-clicking on the following:  
![image](https://github.com/user-attachments/assets/668e4b5e-4233-42fd-9596-902ba7e15036)

A menu similar to this should appear:  
![image](https://github.com/user-attachments/assets/dae16f07-1df1-4deb-b539-92f9493b414a)

In this menu, you will find the option to duplicate the tab.

you should be able to resize it aswell

**Comment by JannisX11** [comment]
You want to duplicate the paint edit tab? By which I guess you mean the Paint panel?
And you want to do that in 3D space? So move parts of the interface into the 3D workspace, as 3D elements? And then those can be used there to model and can be resized?
Sorry, I'm a little confused.

**Comment by aurora-1-2-3** [comment]
yeah i meant duplicating any editing tab. The 3d space was for an example

**Comment by JannisX11** [comment]
I'm gonna interpret that as creating multiple instances of UI panels, OK?

**Comment by aurora-1-2-3** [comment]
yeah this should work
